bedrock/firefox/templates/firefox/whatsnew/exp/whatsnew-fx70-5.html

```diff
@@ -0,0 +1,99 @@
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success show-firefox-desktop-70">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <div class="mzp-c-emphasis-box">
+        <img class="c-emphasis-box-logo" src="{{ static('protocol/img/logos/firefox/logo-md.png') }}" alt="">
+        <h2 class="c-emphasis-box-title">
+          {{ _('10 billion trackers blocked is 10 billion reasons to join Firefox') }}
+        </h2>
+        <p>{{ _('The Firefox browser keeps your internetting safe from invasive companies and advertisers. Now get the same respect for your privacy in the rest of your online life.') }}</p>
+
+        <p class="cta">{{ fxa_cta_link(entrypoint=_entrypoint, entrypoint_experiment=_utm_campaign, entrypoint_variation=_entrypoint_variation, action='signup', button_text='Sign In', button_class='mzp-c-button mzp-t-product mzp-t-small', utm_campaign=_utm_campaign, utm_content='top-cta') }}</p>
+    </div>
+  </section>
+
+  <section class="content-extra">
+    <div class="mzp-l-content">
+        <div class="l-columns-one c-picto-list">
+          <div class="c-picto-block t-adjacent">
+            <div class="c-picto-block-image">
+              <img src="{{ static('img/logos/firefox/logo-lockwise.svg') }}" alt="">
+            </div>
+
+            <div class="c-picto-block-body t-big">
+              <p>{{ _('Manage your passwords with <strong>Firefox Lockwise</strong>.') }}</p>
+            </div>
+          </div>
+
+          <div class="c-picto-block t-adjacent">
+            <div class="c-picto-block-image">
+              <img src="{{ static('img/logos/firefox/logo-monitor.svg') }}" alt="">
+            </div>
+
+            <div class="c-picto-block-body t-big">
+              <p>{{ _('Stay ahead of hackers and breaches with <strong>Firefox Monitor</strong>.') }}</p>
+            </div>
+          </div>
+
+          <div class="c-picto-block t-adjacent">
+            <div class="c-picto-block-image">
+              <img src="{{ static('img/firefox/glyphs/tracking-protection.svg') }}" alt="">
+            </div>
+
+            <div class="c-picto-block-body t-big">
+              <p>{{ _('See it all in action on your <strong>Privacy Protections</strong> dashboard.') }}</p>
+            </div>
+          </div>
+        </div>
+
+        <p class="cta">{{ fxa_cta_link(entrypoint=_entrypoint, entrypoint_experiment=_utm_campaign, entrypoint_variation=_entrypoint_variation, action='signup', button_text='Sign In', button_class='mzp-c-button mzp-t-product mzp-t-small', utm_campaign=_utm_campaign, utm_content='bottom-cta') }}</p>
```

Comment:
This is also true for variations 6 & 7

Reply:
The desire was for a second CTA at the bottom of the pages where there's a lot of content (because The Fold), but I agree this lone "Sign In" at the bottom lacks context. I'd suggest adding some other headline above the button, maybe with some more space to separate it from the other content. I'll see what the team thinks about that.

Reply:
We've opted to change the bottom button to "Get a Firefox Account"